modules/starfish/2.0/config/default.yaml

```diff
@@ -0,0 +1,51 @@
+lcr-modules:
+    
+    starfish:
+
+        dirs: 
+            
+            _parent: "__UPDATE__"
+
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {tumour_id} {normal_id} {pair_status}
+            # Each key:value pair is the name of the variant caller and the path to its VCF file
+            # Each key should be unique in this set. 
+            vcf: 
```

Comment:
Doesn't this tool need at least two variant callers to work? I guess it doesn't matter if they'll get clobbered by the user config but should we put at least two here, e.g. Strelka and Lofreq?

Reply:
In fact, why not list all the ones we currently have available and tell the user to delete/comment out any they are not using?

Reply:
I was _trying_ to find a way to have them blank or commented out in the default config so users don't have to copy the whole default.yaml, but if there wasn't a placeholder here it wouldn't get overwritten by the values in my GAMBL config. I also wanted the "__UPDATE__" value here because it throws a helpful error if the user forgets to set this value. You don't think having one example is good enough?

Reply:
It's probably good enough, but if the minimum will always be 2 why not put two? Notably, what happens if someone decides to use this tool with two or three variant callers that _aren't_ strelka? Does this produce an error that would require them to copy the config anyway?

Reply:
Added some comments to the config to clarify.

Reply:
The demo should clarify how the config should be set up.